File: app/src/main/java/in/ac/bvmengineering/udaan2k19/DataClass/Event.java
package in.ac.bvmengineering.udaan2k19.DataClass;

import android.support.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;

public class Event implements Serializable {

    @SerializedName("tagline")
    private String tagLine;
    @SerializedName("entryFee")
    private int fees;
    @SerializedName("notes")
    private String notes;
    @SerializedName("_id")
    private String id;
    @SerializedName("department")
    private String dept;
    @SerializedName("eventName")
    private String name;
    @SerializedName("eventType")
    private String type;
    @SerializedName("rounds")
    private ArrayList<String> rounds;
    @SerializedName("managers")
    private ArrayList<Manager> managers;
    @SerializedName("teamSize")
    private int teamSize;

    public int getTeamSize() {
        return teamSize;
    }

    public String getNotes() {
        return notes;
    }
    public int getFees() {
        return fees;
    }

    public String getTagLine() {
        return tagLine;
    }

    public String getId() {
        return id;
    }

    public String getDept() {
        return dept;
    }

    public String getName() {
        return name;
    }

    public ArrayList<String> getRounds() {
        return rounds;
    }

    public String getType() {
        return type;
    }

    @NonNull
    @Override
    public String toString() {
        return "Event{" +
                "tagLine='" + tagLine + '\'' +
                ", fees=" + fees +
                ", id='" + id + '\'' +
                ", dept='" + dept + '\'' +
                ", name='" + name + '\'' +
                ", type='" + type + '\'' +
                ", rounds=" + rounds +
                ", managers=" + managers +
                '}';
    }

    public ArrayList<Manager> getManagers() {
        return managers;
    }

}

File: app/src/main/java/in/ac/bvmengineering/udaan2k19/DataClass/Event.kt
package `in`.ac.bvmengineering.udaan2k19.DataClass

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.*


class Event : Serializable {

    @SerializedName("tagline")
    val tagLine: String? = null
    @SerializedName("entryFee")
    val fees: Int = 0
    @SerializedName("notes")
    val notes: String? = null
    @SerializedName("_id")
    val id: String? = null
    @SerializedName("department")
    val dept: String? = null
    @SerializedName("eventName")
    val name: String? = null
    @SerializedName("eventType")
    val type: String? = null
    @SerializedName("rounds")
    val rounds: ArrayList<String>? = null
    @SerializedName("managers")
    val managers: ArrayList<Manager>? = null
    @SerializedName("teamSize")
    val teamSize: Int = 0

    override fun toString(): String {
        return "Event{" +
                "tagLine='" + tagLine + '\''.toString() +
                ", fees=" + fees +
                ", id='" + id + '\''.toString() +
                ", dept='" + dept + '\''.toString() +
                ", name='" + name + '\''.toString() +
                ", type='" + type + '\''.toString() +
                ", rounds=" + rounds +
                ", managers=" + managers +
                '}'.toString()
    }

}
